xknx/io/tunnel.py

```diff
@@ -44,8 +44,10 @@ def __init__(
         self,
         xknx: "XKNX",
         local_ip: str,
+        local_port: int,
```

Comment:
Should we add a default parameter for local_port "0"?
This would mean to move it down. eg
```python
__init__(
    gateway_ip: str,
    gateway_port: int,
    local_ip: str,
    local_port: int = 0,
    route_back: bool = False,
    ...
```